RecoLocalCalo/HcalRecAlgos/src/PulseShapeFitOOTPileupCorrection.cc

```diff
@@ -195,10 +195,15 @@ namespace FitterFuncs{
    }
 
    double PulseShapeFunctor::sigmaSiPMQIE10(double ifC) {
```

Comment:
@davidlange6 
The derivation of those numbers are documented in slide 10-11 of  .pdf added in the PR description.
https://indico.cern.ch/event/570231/contributions/2306435/attachments/1341696/2021200/PR15959.pdf
Do you want me to add a comment in the code ?